cirq/experiments/t2_decay_experiment_test.py

```diff
@@ -297,8 +369,33 @@ def test_bad_args():
                                       delay_sweep=product)
 
 
-def test_str():
+def test_cpmg_circuit():
+    """Tests sub-component to make sure CPMG circuit is generated correctly."""
+    q = cirq.GridQubit(1, 1)
+    t = sympy.Symbol('t')
+    circuit = t2._cpmg_circuit(q, t, 2)
```

Comment:
I understand the sentiment, but I think I will leave these in for now.  It's hard to come up with good noise models that are also testable, which makes testing the output of the experiment tricky.  I think, in this case, it's worth testing the intermediate values of the circuit and sweeps to make sure it is doing the correct experiment, rather than just testing the result of the experiment.

Reply:
Fair enough.